expressjs-backend/routes/listingRoutes.js

```diff
@@ -318,6 +318,29 @@ router.put("/images/:listingID", upload.array('image'), async (req, res) => {
   }
 });
 
+router.get('/api/listings/:listingID', async (req, res) => {
```

Comment:
I believe we already have a route for retrieving a single listing by its listing id, the router.get("/:listingID/") on line 98.

Reply:
Great. I will look. I wasn't sure and I didn't check so I made another. I will reintegrate it. Thanks